server/v2/cometbft/go.mod

```diff
@@ -8,6 +8,7 @@ replace (
 	cosmossdk.io/core/testing => ../../../core/testing
 	cosmossdk.io/server/v2 => ../
 	cosmossdk.io/server/v2/appmanager => ../appmanager
+	cosmossdk.io/server/v2/stf => ../stf
```

Comment:
can this be removed now?

Reply:
no we still need it for tests set up I think